frontend/packages/ux-editor/src/components/config/componentSpecificContent/Summary2/Override/Summary2OverrideDisplaytype.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { StudioCard, StudioNativeSelect } from '@studio/components';
+import type {
+  Summary2OverrideConfig,
+  SummaryCustomTargetType,
+} from 'app-shared/types/ComponentSpecificConfig';
+import { useCustomConfigType } from './hook/useCustomConfigType';
+import { useTranslation } from 'react-i18next';
+
+export interface Summary2OverrideDisplaytypeProps {
+  override: Summary2OverrideConfig;
+  onChange: (override: Summary2OverrideConfig) => void;
+}
```

Comment:
To keep our code consistent, i suggest that we change to `type` here as that is what we do elsewhere 😄 
```suggestion
export type Summary2OverrideDisplaytypeProps = {
  override: Summary2OverrideConfig;
  onChange: (override: Summary2OverrideConfig) => void;
}
```

Reply:
Fixed